config/src/config.rs

```diff
@@ -1491,6 +1499,14 @@ impl DroppedFileQuoting {
             Self::SpacesOnly => s.replace(" ", "\\ "),
             // https://docs.rs/shlex/latest/shlex/fn.quote.html
             Self::Posix => shlex::quote(s).into_owned().to_string(),
+            Self::Windows => {
+                if s.contains(' ') {
+                    format!("\"{}\"", s)
+                } else {
+                    s.to_string()
+                }
+            }
+            Self::DoubleQuoteAlways => format!("\"{}\"", s),
```

Comment:
I'm not sure about this vs. the other methods.  Can we remove it?

Reply:
I see your comment about this being for msys. Since this is for Windows, can this be renamed to `WindowsAlwaysQuoted` or something like that?

Reply:
Sure! `Posix` would be more useful for unix users, so this would be mostly used on Windows.